kubehub/migrations/0001_initial.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.0.2 on 2020-01-25 14:23
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='CloudProvider',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('cp_type', models.CharField(max_length=20)),
```

Comment:
why cp_type, is type reserved keyword?

Reply:
Yes, `type()` function returns class type

Reply:
ok!